k-distribution/src/main/scripts/bin/krun

```diff
@@ -300,10 +303,22 @@ do
       ;;
 
       --pattern)
+      if [ -z "$noPattern" ]; then
       pattern="$2"
+      else
+      error 1 "The option --pattern cannot be used with the option --no-pattern"
+      fi
       shift
       ;;
 
+      --no-pattern)
+      if [ -z "$pattern" ]; then
+      noPattern=true
+      else
+      error 1 "The option --no-pattern cannot be used with the option --pattern"
+      fi
+      ;;
+
```

Comment:
Can these `if` statements be indented? They're tricky to read as they are

Reply:
Sure, my bad